docs/gettingstarted/index.rst

```diff
@@ -95,7 +95,7 @@ Getting started with Flocker
 
          .. code-block:: console
 
-            you@laptop:~$ sudo apt-get update && apt-get install -y gcc python2.7 python-virtualenv python2.7-dev && \
+            you@laptop:~$ sudo apt-get update && sudo apt-get install -y gcc python2.7 python-virtualenv python2.7-dev && \
```

Comment:
Why do we need to include the `apt-update` step? I imagine that might have been included because the author was testing in a docker container, which starts up without apt-update ever having been run.